Script/char.dat

```diff
@@ -4047,16 +4256,57 @@ zombie
     CanRead = true;
     CreateUndeadConfigurations = false;
     DeathMessage = "As @Dd collapses, you hear her hiss: \"Peeeeeetrrrruuuuuuuss!!!\"";
+    HostileReplies =
+    {
+      25,
+      "\"A pity.\"",
+      "\"Are you so easily decieved?\"",
+      "\"You will not suffer... for long.\"",
+      "\"It must be this way.\"",
+      "\"My city... in ruins.\"",
+      "\"This will be quick.\"",
+      "\"How we've fallen.\"",
+      "\"You will fall as we did.\"",
+      "\"This is for the best.\"",
+      "\"He cannot be stopped.\"",
+      "\"He must be stopped.\"",
+      "\"I will save you.\"",
+      "\"I will make you understand.\"",
+      "\"How long have I slumbered?\"",
+      "\"Let us end it.\"",
+      "\"I will stop this.\"",
+      "\"You cannot win.\"",
+      "\"You cannot hurt me.\"",
+      "\"You have been betrayed.\"",
+      "\"I must do this, for your sake.\"",
+      "\"Curse His name!\"",
+      "\"It cannot end like this.\"",
+      "\"I was wrong.\"",
+      "\"Return them to me!\"",
+      "@Dd whispers: \"Please... \"";
+    }
     FriendlyReplies =
     {
-      5,
+      17,
       "\"Once, I was a duchess of a great dwarven fortress-city of Khaz-zadm.\"",
       "\"After the Dwarven Wars, our land lay in ruins. Our mines were undermined and filled with masterless golems. Our resources were stretched thin. That was when Petrus came with his offer of charity. I should have known better.\"",
       "\"Naught but ashes remain from my people. Petrus even saw that the word would not spread to Kharaz-arad or other dwarven lands.\"",
       "\"No one remembers Priscus, the former High Priest of Valpurus, these days. Only me.\"",
+      "\"The poor children of Khaz-zadm! Taken from their murdered parents and trained to die at the behest of the high priest, praising Valpurus with their dying breath.\"",
+      "\"Khaz-zadm once held the richest deposits of valpurian ore on this whole continent. No wonder Attnam has no shortage of vaplurium these days.\"",
+      "\"Attnam and Khaz-zadm were close partners for many years. Attnam bought its sacred metal from us, but we needed attnamese priests to bless the smelting, lest a burnt slag was produced instead of a magical metal.\"",
+      "\"I remember with fondness our shared visits with high priest Priscus. I was greatly saddened by the betrayal of the orcish delegation and his death. Too late did I learn the truth - I fell for the tricks of the same enemy he did.\"",
+      "\"High priest Priscus had a life-long dream - Pax Attnamica, a peace amongst all the lands, races and religions. Alas, not everyone even in his own country agreed with his pacifistic ways and evetually, his efforts led to his untimely demise.\"",
+      "\"Many have recounted the story differently, but I believe this: It was neither the retinue of Priscus, nor the orcish delegation of Al-Khan who betrayed and massacred the other. There is someone else to blame.\"",
```

Comment:
Suggest 'Vul-Khan', or 'Vol-Khan' instead of 'Al-Khan'

Reply:
I rather like Vul-Khan. It's now cannon that orcish assassin use a nerve pinch on their targets.

Reply:
BTW, I wonder how many people will ever get to read this and similar dialogue. :D

Reply:
> Shall we make this change?

Done. :)